packages/blockfrost/src/BlockfrostToOgmios.ts

```diff
@@ -19,9 +19,10 @@ export const BlockfrostToOgmios = {
   // without `as OgmiosSchema.Utxo` above TS thinks the return value is (OgmiosSchema.TxIn | OgmiosSchema.TxOut)[][]
 
   blockToTip: (block: Responses['block_content']): OgmiosSchema.Tip => ({
-    blockNo: block.height,
+    // Review: if height and slot be actually be null we should check it and either throw or return something else
```

Comment:
`slot` and `height` are indeed nullable in Ouroboros Classic (a legacy protocol that we have to consider in a uniform model) since there were special epoch boundary blocks, but these would never be returned for a tip query. We can simply throw this as an exception